File: src/BreakfastProvider.Api/Models/Responses/StaffMemberResponse.cs
namespace BreakfastProvider.Api.Models.Responses;

public record StaffMemberResponse
{
    public int Id { get; init; }
    public string Name { get; init; } = string.Empty;
    public string Role { get; init; } = string.Empty;
    public string Email { get; init; } = string.Empty;
    public bool IsActive { get; init; }
    public DateTime HiredAt { get; init; }
    public DateTime CreatedAt { get; init; }
}

File: java/breakfast-provider-app/src/main/java/io/lemonlion/breakfast/model/response/StaffMemberResponse.java
package io.lemonlion.breakfast.model.response;

import java.time.Instant;

/** Twin of C# {@code StaffMemberResponse}. */
public record StaffMemberResponse(
        int id, String name, String role, String email, boolean isActive, Instant hiredAt, Instant createdAt) {
}
